charts/prometheus-thanos/templates/store-gateway-statefulset.yaml

```diff
@@ -44,7 +44,14 @@ spec:
           - store
           - --data-dir=/data
           - --log.level={{ .Values.storeGateway.logLevel }}
+          {{ if .Values.storeGateway.indexCacheConfig }}
+          - |
+            --index-cache.config=type: {{ .Values.storeGateway.indexCacheType }}
+            config:
+            {{- toYaml .Values.storeGateway.indexCacheConfig | nindent 14 }}
+          {{ else if .Values.storeGateway.indexCacheSize }}
           - --index-cache-size={{ .Values.storeGateway.indexCacheSize }}
```

Comment:
Should we get rid of this config in favour of `index-cache.config`?

Reply:
I thought I'd leae it in to support old values files, but happy to get rid of it if you thing that's better?

Reply:
Updated. 👍